src/Umbraco.PublishedCache.NuCache/Property.cs

```diff
@@ -242,21 +233,17 @@ private CacheValues GetCacheValues(IAppCache? cache)
             return _interValue;
         }
 
-        if (_sourceValues == null)
-        {
-            _sourceValues = new Dictionary<CompositeStringStringKey, SourceInterValue>();
-        }
+        EnsureSourceValuesInitialized();
```

Comment:
Is it ever possible to get here in a state where a null _sourceValues is useful here? This piece of code is very badly documented, but `GetSourceValue` seems to return null for `SourceValue` always when there are no source values passed in ctor?